src/components/GrowlNotification/index.tsx

```diff
@@ -58,13 +73,11 @@ function GrowlNotification(_, ref) {
      * @param {Number} val
      */
     const fling = useCallback(
-        (val = INACTIVE_POSITION_Y) => {
+        (val = INACTIVE_POSITION_Y) =>
```

Comment:
Why were the braces removed? Do we depend on the return value?

Reply:
I was able to remove them here due to there only being one statement in the return of the arrow function, in both implementations the fling animation worked exactly the same.